guides/v2.1/extension-dev-guide/validate/validate.md

```diff
@@ -1,13 +1,10 @@
 ---
 group: php-developer-guide
-subgroup: 06_Validate
 title: Validate
-menu_title: Validate
-menu_order: 1
-menu_node: parent
 ---
 
 After your {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %} is built, you should test it. Read through the Definition of Done to confirm that your module and coding practice is compliant with Magento.
```

Comment:
compliant with Magento guidelines ==>  Would it be useful to link to the Magento guidelines?